brayns/common/scene/Model.h

```diff
@@ -0,0 +1,241 @@
+/* Copyright (c) 2015-2017, EPFL/Blue Brain Project
+ * All rights reserved. Do not distribute without permission.
+ * Responsible Author: Cyrille Favreau <cyrille.favreau@epfl.ch>
+ *
+ * This file is part of Brayns <https://github.com/BlueBrain/Brayns>
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License version 3.0 as published
+ * by the Free Software Foundation.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+#ifndef Model_H
+#define Model_H
+
+#include <brayns/api.h>
+#include <brayns/common/BaseObject.h>
+#include <brayns/common/geometry/Cone.h>
+#include <brayns/common/geometry/Cylinder.h>
+#include <brayns/common/geometry/Sphere.h>
+#include <brayns/common/geometry/TrianglesMesh.h>
+#include <brayns/common/types.h>
+
+SERIALIZATION_ACCESS(ModelTransformation)
+SERIALIZATION_ACCESS(Model)
+SERIALIZATION_ACCESS(ModelDescriptor)
+
+namespace brayns
+{
+/**
+* @brief The ModelTransformation struct defines the translation, rotation and
+* scaling parameters to be applied to a model
+*/
+struct ModelTransformation : public BaseObject
+{
+    ModelTransformation() = default;
+    ModelTransformation(ModelTransformation&& rhs) = default;
+    ModelTransformation& operator=(ModelTransformation&& rhs) = default;
```

Comment:
All constructors/assignment are default, maybe remove all?

Reply:
We cannot because there is another constructor with a different signature below  :(

Reply:
Okay, I missed that :+1: